### ISSUE TITLE
[BUG Report]: LayerNormalization error

### ISSUE THREAD

**Issue by JustDooooIt** [SciSharp/TensorFlow.NET#1216]
### Description
var gradient = g.gradient(y, tensor);report error
After removing LayerNormalization, the program runs normally.
![021ca9f5ad43a44defa93ff05c25d4c7](https://github.com/SciSharp/TensorFlow.NET/assets/43512399/5c12b679-768e-47ac-b502-37ff48849e3e)

### Reproduction Steps

```
[TestMethod]
public void SimVPTest()
{
    var model = new TestModel1();
    using var g = tf.GradientTape();
    var tensor = tf.random.normal((1, 7, 8, 13));
    g.watch(tensor);
    var y = model.Apply(tensor);
    var gradient = g.gradient(y, tensor);
}
public class TestModel1 : Layer
{
    ILayer conv2d;
    ILayer norm;

    public TestModel1() : base(new())
    {
        conv2d = tf.keras.layers.Conv2D(16, 3, 1, "same");
        norm = tf.keras.layers.LayerNormalization(-1);
    }

    protected override Tensors Call(Tensors inputs, Tensors state = null, bool? training = null, IOptionalArgs optional_args = null)
    {
        var x = conv2d.Apply(inputs);
        x = norm.Apply(x);
        return x;
    }
}
```

**Comment by JustDooooIt** [comment]
emmmm,
